Classes/Command/BehatCommandController.php

```diff
@@ -52,8 +52,8 @@ public function setupCommand()
 
         $seleniumBinaryPath = FLOW_PATH_ROOT . 'bin/selenium-server.jar';
         if (!is_file($seleniumBinaryPath)) {
-            $seleniumVersion = 'selenium-server-standalone-2.44.0.jar';
-            $seleniumUrl = 'http://selenium-release.storage.googleapis.com/2.44/' . $seleniumVersion;
+            $seleniumVersion = 'selenium-server-standalone-2.53.1.jar';
+            $seleniumUrl = 'http://selenium-release.storage.googleapis.com/2.53/' . $seleniumVersion;
```

Comment:
@johannessteu why not update to the latest selenium server while we are "upgrading" everything anyway and having a breaking change? Latest current ist {{3.141/selenium-server-standalone-3.141.59.jar}}. Does it work? It seems that Mink/Selenium2Driver is already using WebDriver anyway, which would make it "compatible" also with Selenium Server 3.x, but I have not tested it, and unfortunately it's not really documented anywhere. Maybe you can take a look at it while at it?

Reply:
The latest latest had some issues while starting. But right now the selenium setup itself does not work at all in our pipeline. I'll come up wtih another PR to make it running. Also the selenium seems to be buggy somehow and i'm thinking about chromedriver right now. But yeah, it dies use WebDriver as far as i know!

But i'd like to get this one in before so we can make sure everything at all does work so it's easier to test the selenium stack at all. Guess almost all behat tests in the core won't work as pretty much has changed!

Reply:
But in general it's up to the user to choose a selenium version. You need to execute it in your pipeline/ci manually at all, so it shouldn't stop you to curl/wget the latest version and start that one instead of `bin/selenium-server.jar`, right?

Reply:
You are right with that, we do it also in our docker containers!

Having it fetched in behat:setup is just a convenience for a novice to get it quickly running. And for that it is sufficient to be a "good old version" which simply works.

On a more broad aspect - it would be great to have documented somewhere what component is compatible with which versions, as it is very intransparent for a newcomer (and even for a advanced user) to understand that, as so many things involved:

* neos/behat
* behat/behat
* MinkSelenium2Driver
* selenium-standalone
* Browser in different versions

I consider this lack of documentation a failure of the Behat community, we could at least use the knowledge in the Neos community to have some guidance on our side. 

This is getting offtopic, so let's close this for your change - and keep up the good work so far! Looks promising!

Reply:
Yeah! I'd actually also like to see this package polished and documented! And probably bundled with some more examples etc. We should tackle this topic once we got the update itself done!